src/components/GuardianCard.tsx

```diff
@@ -0,0 +1,61 @@
+import { Gray1, Gray3, Gray4, TextDark, TextLight } from '../colors';
+
+import React from 'react';
+import { rem } from 'polished';
+import styled from 'styled-components';
+
+type Props = {
+  address: string;
+  status: number;
+};
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  /* justify-content: center; */
+  align-items: center;
+  border: 1px solid ${Gray3};
+  width: ${rem(240)};
+  height: ${rem(230)};
+  border-radius: ${rem(16)};
+  background: linear-gradient(to bottom, ${Gray1} 0%, ${Gray1} 64px, #FFFFFF 64px, #FFFFFF 100%);
+`;
+
+const Ellipse = styled.div`
+  border-radius: 50%;
+  background: #EBE4FF;
+  height: ${rem(64)};
+  width: ${rem(64)};
+  margin-top: ${rem(32)};
+`;
+
+const Address = styled.div`
+  margin-top: ${rem(16)};
+  color: ${TextLight};
+  font-weight: 500;
+`;
+
+const StatusArea = styled.div`
+  margin-top: ${rem(16)};
+  color: ${Gray4};
+`;
+
+export const GuardianCard = (props: Props) => {
+  const { address, status } = props;
+
+  const renderStatus = () => {
```

Comment:
rendering via rendering functions has fallen out of favor over the past few yrs. preference from the community (and mine too), is to be as explicit w the code as possible :D 

given 
```typescript
      <StatusArea>
        Pending
      </StatusArea>
```

is so short, might as well just place in the actual `render` :D

Reply:
Ohh interesting, didnt know that.  And how would you handle a conditional render based on status then?  Ex, the statuses are pending, signed, etc?  Have the JS code in there within { } to evaluate what to render?  Still not great at that

Reply:
Because im imagining status as an enum, and it will translate to a rendering of that here, so the icon and string dont need to get passed around

Reply:
ah yea. so you can do something like: 
```jsx
<OuterContainer>
  { currentStatus === Status.Loading && <Loading /> }
  { currentStatus === Status.Signed && <SignedComponent /> }
</OuterContainer>

```

Reply:
So, the right side (component) evaluates to true?  Always I guess, right?